doc/source/user_guide/krylov.rst

```diff
@@ -20,14 +20,16 @@ the response across the frequency range:
 * Solves the reduced system
 * Expands the results back to compute the harmonic response 
 
-MAPDL provides the following ways to implement a harmonic analysis using
-the Krylov method:
+Mechanical APDL provides the following ways to implement a harmonic analysis
+using the Krylov method:
 
-#. Mechanical APDL commands.
-#. APDL macros as described in "Frequency-Sweep Harmonic Analysis via the
-   Krylov Method" in the *Structural Analysis Guide*.
-#. PyMAPDL. You can use the Python programming language to expose the Krylov
-   features as described in this section.
+#. Mechanical APDL commands
```

Comment:
I got a feedback from Jeff s team to check if we could modify this portion as follows, I would like your opinion on this:

Mechanical APDL provides the following ways to implement a harmonic analysis
using the Krylov method:
#. Mechanical APDL commands.
#. APDL macros as described in "Frequency-Sweep Harmonic Analysis via the
   Krylov Method" in the *Structural Analysis Guide*.

  Now we can also use Pymapdl to access the Krylov feature which is described in this section.

Reply:
To me "two ways" should not be correct... Like Macros are just MAPDL commands grouped together.... it is the same underlying thing..... but yeah... let's roll with it. xD

Reply:
@kmkoshy 
I am fine with everything but the "Now we" that starts the last sentence for two reasons:

1. The goal of documentation is to be timeless--which means no reference to now, currently, future, past.
2.  The style says to use singular "you" and not plural "we".

There are also a few other style errors. I'd suggest keeping this sentence parallel in structure to the one introducing the list. My recommended rewrite follows.

"PyMAPDL also provides a way to implement a harmonic analysis using the Kylov method. Subsequent sections describe how to use the Krylov method in PyMAPDL.

In doing this, I'd bump the "Assumptions section up to the same level as the "Kylov method implementation in PyMAPDL" heading. The text says that they are assumptions when using the Krylov PyMAPDL method. If the assumptions are also applicable to the Mechanical APDL implementation, I'd not make them their own section but just tag the word Assumptions in bold.

Reply:
@PipKat modified as per your suggestion.
Assumptions are applicable for Mechanical APDL implementation as well as PyMAPDL so keeping it as before, But modified the title to make it bold.